v2/model_apply.py

```diff
@@ -0,0 +1,100 @@
+"""This file applies the most recent model to incoming data."""
+import re
+import os
+import json
+import datetime
+import pandas as pd
+
+# View everything:
+pd.options.display.max_rows = 100
+pd.options.display.max_columns = 100
+
+# Machine Learning externals
+import joblib
+
+# Load custom functions
+from user_reports_preprocessor import pre_process_data
+
+# ==============================================================================
+# Begin
+# ==============================================================================
+
+# Get newest model
+all_dirs = os.listdir("models")
+most_recent = max(all_dirs, key=lambda d: datetime.datetime.strptime(d, '%m-%d-%Y'))
+most_recent = os.path.join("models",most_recent)
```

Comment:
For now, this is good. We should plan to use a s3 bucket to store and get the latest model.

Reply:
Agreed. For now, is storing locally. In next iteration, will do. Let me add this as an issue.

Reply:
Added as an issue #9